### PR TITLE
fix(windows): bootstrap installer not signed

### LINKED ISSUE #4066: bug(windows): bootstrap installer does not have a digital signature on downloads.keyman.com
**Describe the bug**

The bootstrap installer that is downloadable from keyman-staging.com for Windows installation does not include a digital signature.

**To Reproduce**

1. Go to https://keyman-staging.com/keyboards/install/khmer_angkor
2. Click on 'Download keyboard'
3. See error

**Expected behavior**

The downloaded installer should be digitally signed.

**Screenshots**

image

---

**Keyman for Windows/macOS/Linux:**
 - OS: Windows 10 x64 1909 
 - Keyman version: 14.0.195-alpha

### PR DESCRIPTION
**Pull request by mcdurdin**
Fixes #4066.

### CODE REVIEW

**Comment by mcdurdin** [comment]
Okay, validated:

1. setup.exe is signed in the artifacts (correct).
2. setup.exe is *not* signed in the keyman developer redist (correct).
3. bundled installer is signed and not broken.

File: windows/src/developer/inst/kmdev.wxs [hunk 1]
```diff
@@ -256,7 +256,7 @@
       <!-- Redistributable files -->
 
       <Component Directory="dirRedistSetup">
-        <File Name="setup.exe" Source="..\..\..\bin\desktop\setup.exe" KeyPath="yes" />
+        <File Name="setup.exe" Source="..\..\..\bin\desktop\setup-redist.exe" KeyPath="yes" />
```
Comment (darcywong00) [inline windows/src/developer/inst/kmdev.wxs:259]: I take it `Name` doesn't need to change to "setup-redist.exe"

**Review by darcywong00 — APPROVED** [review]
lgtm

File: windows/src/developer/inst/kmdev.wxs (hunk 1 above)
Comment (mcdurdin) [reply windows/src/developer/inst/kmdev.wxs:259]: Correct. In theory, we could do that, but it means more changes to other locations in the build.

**Pull request merged**